pairing/config.go

```diff
@@ -0,0 +1,9 @@
+package pairing
+
+type Config struct {
```

Comment:
The paring config should be moved to handler.Options no need for a separate package with only one config struct.